packages/integers_stubs_js/integers_stubs_js.1.0~alpha-repo/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Javascript stubs for the integers library in js_of_ocaml"
+description: "Javascript stubs for the integers library in js_of_ocaml."
+maintainer: "opensource@o1labs.org"
+authors: "O(1) Labs, LLC <opensource@o1labs.org>"
+license: "MIT"
+homepage: "https://github.com/o1-labs/integers_stubs_js"
+bug-reports: "https://github.com/o1-labs/integers_stubs_js/issues"
+depends: [
+  "js_of_ocaml" {>= "3.6.0"}
+  "zarith_stubs_js"
+  "dune" {>= "1.6"}
+  "integers" {with-test & >= "0.6.0"}
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+dev-repo: "git+https://github.com/o1-labs/integers_stubs_js.git"
+url {
+  src: "https://github.com/o1-labs/integers_stubs_js/archive/1.0.tar.gz"
```

Comment:
This is the same as the upstream package, isn't it?

Reply:
Yes

Reply:
It wants to point to your branch, though - `https://github.com/novemberkilo/integers_stubs_js/archive/master.tar.gz` (no checksum needed)

Reply:
The mainline repo should build with the updated js_of_ocaml branch so we shouldn't need my fork here?

Reply:
I'm just fixing a test related bug on my branch